src/server/plugins/requestLogging.js

```diff
@@ -28,16 +29,23 @@ TIMERS
 Why symbols?
 It prevents conflicts with existing keys, native or from other plugins
 */
+export const $ExternalRequestDuration = Symbol('$ExternalRequestDuration');
 export const $RequestFullDuration = Symbol('$RequestFullDuration');
 export const $RequestOverhead = Symbol('$RequestOverhead');
 export const $RouteHandler = Symbol('$RouteHandler');
 export const $ResponseBuilder = Symbol('$ResponseBuilder');
 
 const passThrough = ({ log }) => log;
+
 const UTILS = {
   configureRequestLog: passThrough,
 };
 
+function formatProtocol(parsedUrl) {
+  const { protocol } = parsedUrl;
+  return protocol.replace(/:$/, '');
+}
```

Comment:
```suggestion
const formatProtocol = ({ protocol }) => protocol.replace(/:$/, '');
```

1. We use arrow functions everywhere, at least in this file
2. Less code

Reply:
@giulianok arrow functions aren't recommended by Fastify. You can get better error messages with named functions, and `this` is functional within a component.

Reply:
It also make performance profiling much easier to understand. Ideally we should be heading in the opposite direction.